osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -107,8 +96,28 @@ private void load(DirectOverlay direct, SettingsOverlay settings, OsuConfigManag
                     }
                 },
                 sideFlashes = new MenuSideFlashes(),
+                songTicker = new SongTicker
+                {
+                    Anchor = Anchor.TopRight,
+                    Origin = Anchor.TopRight,
+                    Margin = new MarginPadding { Right = 15, Top = 5 }
+                }
             });
 
+            if (host.CanExit)
+            {
+                AddInternal(exitConfirmOverlay = new ExitConfirmOverlay
+                {
+                    Action = () =>
+                    {
+                        if (holdDelay.Value > 0)
+                            confirmAndExit();
+                        else
+                            this.Exit();
+                    }
+                });
+            }
+
```

Comment:
You broke `ButtonSystem`'s `OnPressed` when moving this.

Reply:
aaaaaaaa i forgot to push 
![image](https://user-images.githubusercontent.com/191335/73054158-061cf400-3ecd-11ea-9eb3-5149c795bfe7.png)